api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java

```diff
@@ -319,7 +319,8 @@ public void setUp() throws Exception {
             .toolDockerImage(TOOL_DOCKER_IMAGE)
             .autopauseThreshold(AUTOPAUSE_THRESHOLD)
             .dataprocConfig(dataprocConfig)
-            .createdDate(createdDate);
+            .createdDate(createdDate)
+            .errors(Collections.emptyList());
```

Comment:
My guess is that this change is needed because `errors` is now `required` in the Leo entity.  I confirmed that our UI treats null and emptyList in the same way here.